examples/stress_tests/xr_cubes_bench.rs

```diff
@@ -0,0 +1,150 @@
+//! A cube-spawning benchmark.
+//!
+//! Press the arrow keys to adjust the number of cubes, then press R to spawn
+//! them.
+
+use bevy::diagnostic::{FrameTimeDiagnosticsPlugin, LogDiagnosticsPlugin};
+use bevy::prelude::*;
+
+#[derive(Component)]
+struct Cube;
+
+#[derive(Resource)]
+struct Spawner {
+    pub width: i32,
+    pub height: i32,
+    pub cube_size: f32,
+    pub should_spawn: bool,
+    pub material: Option<Handle<StandardMaterial>>,
+    pub mesh: Option<Handle<Mesh>>,
+}
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
```

Comment:
We should use the same resolution / overridden scale factor as other stress tests.